test/e2e/resources_test.go

```diff
@@ -87,31 +87,39 @@ func TestCustomResourcesLimits(t *testing.T) {
 		return client.Do(req)
 	}
 
-	pokeCowForMB := func(mb int) error {
+	bloatAndCheck := func(mb int, wantSuccess bool) {
+		expect := "failure"
+		if wantSuccess {
+			expect = "success"
+		}
+		klog.V(2).Infof("Bloating by %d MB and expecting %s.\n", mb, expect)
 		u, _ := url.Parse(endpoint.String())
 		q := u.Query()
 		q.Set("bloat", fmt.Sprintf("%d", mb))
 		u.RawQuery = q.Encode()
 		response, err := sendPostRequest(test.ServingFlags.ResolvableDomain, u)
 		if err != nil {
-			return err
-		}
-		if response.StatusCode != http.StatusOK {
-			return fmt.Errorf("StatusCode = %d, want %d", response.StatusCode, http.StatusOK)
+			klog.V(5).Infof("Received error '%+v' from sendPostRequest (may be expected)\n", err)
```

Comment:
Not sure I see the value in logging the error if the error is expected. Why not put this down in the "wantSuccess" statement?